app/controllers/api/functions.php

```diff
@@ -990,22 +1303,11 @@
             throw new Exception(Exception::BUILD_NOT_FOUND);
         }
 
-        if ($build->getAttribute('status') !== 'failed') {
```

Comment:
Don't we need this condition?

Reply:
The build is started after uploading the code to storage, right?

Reply:
We dont. Logic for "Create Build" was changed. Previously it was restarting existing deployment. Now we make new deployment.

Reply:
Also this check was not allowing you to redeploy build that was successful. Which is limitation we dont want. For example when you change env variables and want to redeploy - it might have nothing to do with build status.